tests/integration/components/todo-item-test.js

```diff
@@ -0,0 +1,33 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import Ember from 'ember';
+
+moduleForComponent('todo-item', 'Integration - Component - Todo Item', {
+  integration: true
+});
+
+test('completed checkbox triggers a save', function (assert) {
+  var store = this.container.lookup('service:store');
+  var todo;
+
+  Ember.run(function () {
+    todo = store.createRecord('todo', {
+      title: 'Confirm save',
+      isCompleted: false
+    });
+  });
+
+  this.set('todo', todo);
+  this.render(hbs`{{todo-item todo=todo}}`);
+
+  assert.notOk(todo.get('isSaving'), 'todo is not saving before click');
+
+  this.$('input[type=checkbox]').click();
```

Comment:
this line should be wrapped in a `run` (soon we will expose the acceptance test `click` helpers for component tests, but until then a run is appropriate.